src/Recruit/SFA.DAS.Recruit.Api/Models/GetProviderAddress.cs

```diff
@@ -15,12 +15,12 @@ public class GetProviderAddress
         {
             return new GetProviderAddress
             {
-                Address1 = source.Address1,
-                Address2 = source.Address2,
-                Address3 = source.Address3,
-                Address4 = source.Address4,
-                Town = source.Town,
-                Postcode = source.Postcode
+                Address1 = source?.AddressLine1,
```

Comment:
Can source be null ever? if it can be can we check that first instead of having null check on each assignment?

Reply:
Yes, there are one or two providers out of the hundreds with no address details.  I'll have a look and rework if needed.